lib/dockerize_rails/docker_commands/docker_helpers.rb

```diff
@@ -0,0 +1,81 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerHelpers
+      NAMES = {
+        image: {
+          suffix: ":#{DRConfig.application_env}",
+          rails: "#{DRConfig.application_name}_rails".freeze,
+          mysql: "#{DRConfig.application_name}_mysql".freeze,
+          postgres: "#{DRConfig.application_name}_postgres".freeze
+        }.freeze,
+        container: {
+          suffix: '',
+          rails: "#{DRConfig.application_name}_rails_container".freeze,
+          mysql: "#{DRConfig.application_name}_mysql_container".freeze,
+          postgres: "#{DRConfig.application_name}_postgres_container".freeze
+        }.freeze
+      }.freeze
+
+      def self.get_name(service, type)
+        "#{NAMES[type][service]}#{NAMES[type][:suffix]}".freeze
+      end
+
+      def self.services_from_docker_compose
+        docker_compose = YAML.load_file "#{PATHS.current}/docker-compose.yml"
+        docker_compose['services']
+      end
+
+      def self.recurse_merge(first, second)
+        first.merge(second) do |_, first_element, second_element|
+          if first_element.is_a?(Hash) && second_element.is_a?(Hash)
+            recurse_merge(first_element, second_element)
+          else
+            [*first_element, *second_element]
+          end
+        end
+      end
+
+      # rubocop:disable Metrics/AbcSize
+      # rubocop:disable Metrics/MethodLength
+      def self.build_options(definitions, service)
+        options = DockerOptions.new
+        options.image get_name(service, :image)
+        options.name get_name(service, :container)
+        options.hostname '0.0.0.0'
+        definitions['expose'].each {|expose| options.expose expose} if definitions.key?('expose')
+        definitions['environment'].each {|env| options.add_env env} if definitions.key?('environment')
```

Comment:
__[rubocop]__
* Space between { and | missing.
* Space missing inside }.

<!-- Comment by [SideCI](https://sideci.com) -->